fineract-provider/dependencies.gradle

```diff
@@ -80,9 +80,10 @@ dependencies {
 
             'org.drizzle.jdbc:drizzle-jdbc',
 
-            'org.apache.poi:poi',
-            'org.apache.poi:poi-ooxml',
-            'org.apache.poi:poi-ooxml-schemas',
+            'org.apache.poi:poi:4.1.2',
+            'org.apache.poi:poi-ooxml:4.1.2',
```

Comment:
I also noticed some of these dependencies were already defined in build.gradle though just older, so you could just go and update the version numbers there. Is that OK with you??